<launch>    
    <node pkg="stage_ros" type="stageros" name="stageros" args="$(find simulation_2017)/worlds/mining_field_4multibots.world" respawn="false" output="screen" >
        <!--<remap from="/cmd_vel" to="/robot_0/cmd_vel"/>
        <remap from="/odom" to="/robot_0/odom"/>-->
    </node>

    <node pkg="simulation_2017" type="magic_odomFromStage_4.py" name="magic_odomFromStage_4">
    </node>
    
    <group ns="/robot_0">
        <remap from="/cmd_vel" to="/robot_0/cmd_vel" />
        <remap from="/odom" to="/robot_0/odom" />
        <remap from="/miniBot_poses" to="/robot_0/miniBot_poses" />

        <node name = "minibot_navigator" pkg="simulation_2017" type="minibot_navigator.py">
        </node>
    </group>

    <group ns="/robot_1">
        <remap from="/cmd_vel" to="/robot_1/cmd_vel" />
        <remap from="/odom" to="/robot_1/odom" />
        <remap from="/miniBot_poses" to="/robot_1/miniBot_poses" />

        <node name = "minibot_navigator" pkg="simulation_2017" type="minibot_navigator.py">
        </node>
    </group>

    <group ns="/robot_2">
        <remap from="/cmd_vel" to="/robot_2/cmd_vel" />
        <remap from="/odom" to="/robot_2/odom" />
        <remap from="/miniBot_poses" to="/robot_2/miniBot_poses" />

        <node name = "minibot_navigator" pkg="simulation_2017" type="minibot_navigator.py">
        </node>
    </group>

    <group ns="/robot_3">
        <remap from="/cmd_vel" to="/robot_3/cmd_vel" />
        <remap from="/odom" to="/robot_3/odom" />
        <remap from="/miniBot_poses" to="/robot_3/miniBot_poses" />

        <node name = "minibot_navigator" pkg="simulation_2017" type="minibot_navigator.py">
        </node>
    </group>

</launch>
